modules/commands.py

```
@@ -0,0 +1,442 @@
+    try:
+        from matrix_controller import WebMatrixController
+
+        controller = WebMatrixController()
+        controller.run()
+    except ImportError as e:
+        print(f"❌ Error importing controller: {e}")
+        print(
+            "Make sure all dependencies are installed: pip install -r requirements.txt"
+        )
+        return False
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)